hack/tools/install-cert-mgr.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+while [[ $(kubectl get pods -l app=webhook -n cert-manager -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]]; do echo "waiting for pod" && sleep 5; done
```

Comment:
Is the wait necessary here? Is there a reason we can't just apply the manifest and then let kubernetes' eventual consistency catch up?

Reply:
The creation of the ValildatingWebHookConfiguration is rejected until the certification manager webhook pod becomes ready.  It takes about 30 seconds after the install in my environment.  So the apply manifests fails without this.

Reply:
I remembered there's a `kubectl` command that makes this easier:
```suggestion
kubectl wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook
```

Reply:
It's nice.  I'll change and move it in the Make file.  I'll add --timeout=60s.  Thanks!